src/java/org/apache/cassandra/cql3/statements/AlterKeyspaceStatement.java

```diff
@@ -60,6 +65,9 @@ public void validate(ClientState state) throws RequestValidationException
         if (SchemaConstants.isLocalSystemKeyspace(ksm.name))
             throw new InvalidRequestException("Cannot alter system keyspace");
 
+
```

Comment:
I wasn't able to trigger an automatic formatter for cassandra code, and if I did the refactor with IDEA it would change a lot of unrelated lines so, yes, this could happen.